user_testing.md

```diff
@@ -0,0 +1,23 @@
+Author: Dan Gunter <dkgunter@lbl.gov>
+
+Created: 23 December 2015
+
+# User testing
+
+User testing, also called "usability testing", is where one user at a time is shown something (e.g. a web site, or API) and asked to either (a) figure out what it is, or (b) try to do a typical task. That sentence is stolen directly from this excellent chapter on how to do lightweight user testing:
+
+[Don't Make me Think, Chapter 9: Usability testing on 10 cents a day](http://sensible.com/downloads/DMMTchapter09_for_personal_use_only.pdfΩ)
```

Comment:
You have a stray unicode character at the end of the link, right after the pdf letters.